feat: add accessor protocol support and refactor `stats/base/nanmin` (#6211)
stdlib-js/stdlib: lib/node_modules/@stdlib/stats/base/nanmin/README.md

var nanmin = require( '@stdlib/stats/base/nanmin' );
```

#### nanmin( N, x, strideX )

> Computes the minimum value of a strided array `x`, ignoring `NaN` values.

Reviewer: ```suggestion
Computes the minimum value of a strided array, ignoring `NaN` values.
```

```javascript
var Float64Array = require( '@stdlib/array/float64' );

var x0 = new Float64Array( [ 2.0, 1.0, 2.0, -2.0, -2.0, NaN, NaN, 4.0 ] );
var x1 = new Float64Array( x0.buffer, x0.BYTES_PER_ELEMENT*1 ); // start at 2nd element

var v = nanmin( 4, x1, 2 );
// returns -2.0
```

#### nanmin.ndarray( N, x, strideX, offsetX )

Computes the minimum value of a strided array, ignoring `NaN` values and using alternative indexing semantics.

```javascript
var x = [ 1.0, -2.0, NaN, 2.0 ];

var v = nanmin.ndarray( 4, x, 1, 0 );
// returns -2.0
```

The function has the following additional parameters:

-   **offsetX**: starting index for `x`.

> While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying buffer, the offset parameters support indexing semantics based on a starting indices. For example, to calculate the minimum value for every other value in `x` starting from the second value

Reviewer: ```suggestion
While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying buffer, the offset parameter support indexing semantics based on a starting index. For example, to calculate the minimum value for every other element in `x` starting from the second element
```

> While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying buffer, the offset parameter support indexing semantics based on a starting index. For example, to calculate the minimum value for every other element in `x` starting from the second element

Reviewer: ```suggestion
While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying buffer, the offset parameter supports indexing semantics based on a starting index. For example, to calculate the minimum value for every other element in `x` starting from the second element
```

<!-- eslint no-undef: "error" -->

```javascript
var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
var nanmin = require( '@stdlib/stats/base/nanmin' );

> var x = discreteUniform( 10, -50, 50, {

Reviewer: How you refactored this example does not preserve the original logic. See, e.g., https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/strided/dnanmax#examples
